landoapi/app.py

```diff
@@ -54,6 +54,9 @@ def load_config() -> dict[str, Any]:
         "MAIL_USE_TLS": bool(os.getenv("MAIL_USE_TLS")),
         "SQLALCHEMY_DATABASE_URI": os.getenv("DATABASE_URL"),
         "SQLALCHEMY_TRACK_MODIFICATIONS": False,
+        "APP_SPEC": "treestatus.yml"
+        if os.getenv("TREESTATUS_APP") is not None
```

Comment:
This should be explicit, i.e.,
```suggestion
        if os.getenv("TREESTATUS_APP") == "1"
```

Otherwise values like "0" and "false" will also behave in the same way.